### ISSUE TITLE
Update hooks.md 

### ISSUE THREAD

**Issue by stevensolarz** [stephenafamo/bob#317]
`website/docs/models/hooks.md` is out of date
- remove BeforeUpsertHooks and AfterUpsertHooks (lines 20:21)
- Update 'Registering hooks' section (line 39)
```
## Registering hooks

A hook can be registered with the `AppendHooks` method:

userTable.BeforeUpdateHooks.AppendHooks(myHook)
```

I'd submit a PR but I don't have permission!

**Comment by stephenafamo** [comment]
Yes, unfortunately, several parts of the docs are now out of date.

You can send in a PR by creating a fork and making changes on your fork. Any help is appreciated.
